src/WCSimPhysicsList.cc

```diff
@@ -522,66 +551,55 @@ void WCSimPhysicsList::ConstructHad()
 	  // Choice defined in jobOptions.mac, which is
 	  // read in before initialization of the run manager.
 	  // In the absence of this file, BINARY will be used.
```

Comment:
Once you change WCSimPhysicsMessenger.cc, BERTINI will be the default, so this comment should be updated.